training-react/login/src/app/components/App/components/Login/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
This component shouldn't be in folder `components` because is a screen of `App` so move it to `screen` folder